bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ClientConfiguration.java

```diff
@@ -101,6 +101,31 @@
     public final static String CLIENT_ROLE_SYSTEM = "system";
 
     /**
+     * Use SSL while talking to Bookies
+     */
+    protected final static String USE_SSL = "useSSL";
+
+    /**
+     * Verify SSL certificates
+     */
+    protected final static String VERIFY_SSL_CERTIFICATES = "verifySSLCertificates";
+
+    /**
+     * Enable client authentication using an SSL certificate
+     */
+    protected final static String CLIENT_SSL_AUTHENTICATION = "clientSSLAuthentication";
+
+    /**
+     * Key store which holds the client-side SSL certificate. The certificate must be in PKCS12 format
+     */
+    protected final static String CLIENT_SSL_KEY_STORE = "clientSSLKeyStore";
```

Comment:
For local CA signed certificates, should we also have an option to provide the trustStore along with client/bookie keyStore?